config/default-amo.js

```diff
@@ -24,9 +24,24 @@ module.exports = {
   },
   enableTrailingSlashesMiddleware: true,
   fxaConfig: 'amo',
+
+  // This needs to be kept in sync with addons-server's SUPPORTED_NONLOCALES
+  // settings value: https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L274
+  validLocaleUrlExceptions: [
+    'contribute.json',
+    'google1f3e37b7351799a5.html',
+    'robots.txt',
+    'services',
+    'downloads',
+    'blocklist',
```

Comment:
Since this isn't in addons-server maybe you should add a comment that it exists to support a redirect to the other host?